### DISCUSSION TITLE
Delta between records

### DISCUSSION THREAD

**Discussion by BpStef** [thepirat000/Audit.NET#648] [Q&A]
I'm using Audit.NET EntityFramework Core to save the states of records at a certain point in time.
So it saves all values of a certain record and some extra data like the date, the user, transactionId and action.
Now we want to create an endpoint that returns the changes between those records. Is there an easy way to do this?

**Comment by thepirat000** [comment]
The data providers do not have a standardized method for querying audit events, which falls beyond the scope of this library. 

However, specific data providers may offer custom methods to retrieve an audit event by ID or expose particular query mechanisms, contingent upon the manner in which the events are stored. 

If using EF or any database to store events, creating an endpoint for the query should be straightforward.
